helm/athena/templates/ingress.yaml

```diff
@@ -10,11 +10,11 @@ metadata:
   labels:
     {{- include "athena.labels.common" . | nindent 4 }}
   annotations:
-    {{- if .Values.services.athena.letsencrypt}}
-    kubernetes.io/tls-acme: "true"
```

Comment:
this is old annotations no longer necessary if the `cluster-issuer` annotations is used